### DISCUSSION TITLE
Iceberg table delete support

### DISCUSSION THREAD

**Discussion by nikola-arcadis** [apache/doris#40342] [A - General / Q&A]
I can see that Doris 3.0 supports equality and position delete in Iceberg table but when I try to delete all the rows containing in a single parquet file, I get the error message.
```
SQL: delete from tasks2 where id > 0;
Message: detailMessage = delete command could be only used on olap table
```
I'm wondering what kind of delete support Doris has and will it support row level delete?

**Comment by ixzc** [comment]
it's not support now, maybe at least Q4
